alivee/utils/requests/zillow/zillow_deep_search_result_request_meta.py

```diff
@@ -0,0 +1,24 @@
+from utils.requests.zillow.zillow_base_request_meta import ZillowBaseRequestMeta
+
+
+class ZillowDeepSearchResultsRequestMeta(ZillowBaseRequestMeta):
+    '''
+    Request metadata for zillow GetDeepSearch-Results API:
+    https://www.zillow.com/howto/api/GetDeepSearchResults.htm
+    '''
+    PATH = 'howto/api/GetDeepSearchResults.htm'
+
+    def __init__(self,
+                 service_id,
+                 address,
+                 city=None,
+                 state=None,
+                 zipCode=None,
+                 rentEstimate=False):
+        params = {
+            'address': address,
+            'citystatezip': self.getArea(city, state, zipCode),
+            'rentzestimate': rentEstimate
+        }
+        super(ZillowDeepSearchResultsRequestMeta, self).\
+            __init__(service_id, self.PATH, params)
```

Comment:
How many characters in a line you are limiting now? I'm using 120 now..... (so `\` is not needed for this case)